test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -1056,7 +1056,9 @@ BOOST_AUTO_TEST_CASE(modifier_overrides_function)
 		contract A { modifier mod(uint a) { _; } }
 		contract B is A { function mod(uint a) { } }
 	)";
-	CHECK_ERROR(text, TypeError, "");
+	// Error: Identifier already declared.
```

Comment:
Because two error messages are shown.  I have no expectation about which one comes first and be searched.